articles/explanation-of-error-unrecognized-transaction-id-transaction-may-have-timed-out-and-been-rolled-back.adoc

```diff
@@ -3,7 +3,7 @@
 :zendesk-id: 216757677
 :author: Dana Canzano
 :tags: cypher,http
-:public: 3.0,3.1,3.2,3.3,3.4,3.5
+:neo4j-versions: 3.0,3.1,3.2,3.3,3.4,3.5
```

Comment:
I guess it was a mistake, right?

Reply:
Yeah, I think this looks like a mistake when we were updating attributes. :)